src/popup/components/TransactionInfo.vue

```diff
@@ -14,14 +14,18 @@
     </div>
 
     <div class="parties">
-      <Avatar v-bind="sender" />
+      <Avatar
+        v-bind="sender"
+        :name="punycodeToName(sender.name)"
+      />
       <div class="mid">
         <TriangleRight class="triangle" />
         <div class="line" />
       </div>
       <Avatar
         v-if="recipient.address"
         v-bind="recipient"
+        :name="punycodeToName(recipient.name)"
```

Comment:
This is something that should be on the sdk side, it is the right way to encode names in not english?

Reply:
Yes, it's encoded in accordance with the [documentation](https://docs.aeternity.com/protocol/AENS.html#name)